systemd/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 install(FILES pileft-quirks.service piright-quirks.service hdmi-disable.service
   DESTINATION /lib/systemd/system
 )
+
```

Comment:
```suggestion
```

Reply:
Just drop changes in this file, because you do not have to touch it for this commit.